core/domain/draft_upgrade_services.py

```diff
@@ -88,6 +88,21 @@ def try_upgrading_draft_to_exp_version(
 class DraftUpgradeUtil(python_utils.OBJECT):
     """Wrapper class that contains util functions to upgrade drafts."""
 
+    @classmethod
+    def _convert_states_v32_dict_to_v33_dict(cls, draft_change_list):
```

Comment:
#8718 also seems to migrate to the same version, so depending on what gets merged first, this may need to be changed.